automation/scripts/github_branch_autosync/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why is an Apache license file needed here, there's already one in the repo root?